documentation/Functions.md

```diff
@@ -350,20 +350,32 @@ If the tutorial should be tested on the console environment, you have to specify
 ***
 
 ### displayContent <a name="displayContent"></a>
+This function is only used when you want to display content such as text, image or any file content in your tutorial.
+
 #### parameter
-1. The title of the step. 
-2. An array of json objects with files, content, or images to be rendered within the Katacoda step. The use for this function is to display an image and some descriptive text. No Katacoda syntax is allowed in the files or the content!
-3. (Optional) Path to the current directory where the user is located (relative to the workspace directory). Only needed if the directory is changed within this step.
+you need two parameters for this function.
+
+Example: displayContent(1st parameter, 2nd parameter) 
+
+1st parameter i.e. The title of the step. 
+
+Note: The title should never be empty and it is of type string.
+
+2nd parameter i.e. An array of json objects with files, content, or images to be rendered within the Katacoda step. The use for this function is to display an image and some descriptive text. No Katacoda syntax is allowed in the files or the content!
+
 #### example 
-display("Step title", [{ "file": "files/description.asciidoc" }, { "content": "This is just plain content." }, { "image": "files/image.png" }])
+displayContent("Step title", [{ "file": "files/description.asciidoc" }, { "content": "This is just plain content." }, { "image": "files/image.png" }])
 
 #### Details
-Available attributes in the json objects:
+You can use 3 attributes in the json objects array in 2nd parameter:
+
+2nd parameter = [{1st attribute}, {2nd attribute}, {3rd attribute}]
+
+1st attribute i.e.  "file": Path to a file whose content is to be displayed in the Katacoda step (e.g. .asciidoc or .txt file). The file should be following the formating of asciidoc files. 
 
-1. file: Path to a file whose content is to be displayed in the Katacoda step (e.g. .asciidoc or .txt file). The file should be following the formating of asciidoc files. 
-2. content: Plain text to be displayed in the Katacoda step. This Text should be following the formating of asciidoc files.
-3. image: Path to an image to be displayed in the Katacoda step.
+2nd attribute i.e. "content": Plain text to be displayed in the Katacoda step. This Text should be following the formating of asciidoc files.
 
+3rd attribute i.e. "image": Path to an image to be displayed in the Katacoda step.
```

Comment:
Here you can mention that the image should be located in a subfolder of the playbook directory